Gems/HttpRequestor/Code/Include/HttpRequestor/HttpTextRequestParameters.h

```diff
@@ -34,50 +49,76 @@ namespace HttpRequestor
         TextParameters(TextParameters&&) = default;
         TextParameters& operator=(TextParameters&&) = default;
 
-        //returns the URI in string form as an recipient of the HTTP connection
-        const Aws::String& GetURI() const { return m_URI; }
+        //! Get the URI in string form as an recipient of the HTTP connection.
+        const Aws::String& GetURI() const
+        {
+            return m_URI;
+        }
 
-        //returns the method of which the HTTP request will take. GET, POST, DELETE, PUT, or HEAD
-        Aws::Http::HttpMethod GetMethod() const { return m_method; }
+        //! Get the HTTP method configured to use for a request.
+        Aws::Http::HttpMethod GetMethod() const
```

Comment:
Nit: Same here. Comment for the return value?

Reply:
@sptramer asked me to remove the return value documentation as it added nothing for simple types.